src/resources/Alteration/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './BenefitCompanyResource'
-export * from './LimitedCompanyResource'
-export * from './CommunityContributionCompanyResource'
-export * from './UnlimitedLiabilityCompanyResource'
+export * from './BC'
+export * from './BEN'
+export * from './CCC'
+export * from './ULC'
```

Comment:
I didn't see any reason to be verbose here -- the folder structure already tells us what this is.

Another way to look at is: if this data came from a db, what would this object be called? Probably the simplest thing we could come up with :)